pyrsa/vis/mds_plot.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Definition of visualization methods
+
```

Comment:
Since this doesn't contain a plot, and only transformation functions, maybe this should be clear in the name of the module. Perhaps there should even be a transformations sub package

Reply:
retitled inside to be more precise. BTW, I actually prefer mds_plot, since we have this question in an early PR (vis-dev) whether we should include exact MDS plotting etc., and the answer is not clear yet. Therefore, if additional plotting functions emerge, we can include them in mds_plot.py, similar to the conventions in rdm_plot.py and model_plot.py.